### PR TITLE
Lock version of generator, test failing

### PR DESCRIPTION
**Pull request by purplecabbage**
## Description

For https://jira.corp.adobe.com/browse/ASSETS-33316

Fixes # (issue)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: test/index.test.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ describe("integration tests", function() {
             aio asset-compute test-worker
         `);
     }).timeout(600000);
-    it("should install version 8.3.0 of aio-cli and run developer experience", async function() {
+    it.skip("should install version 8.3.0 of aio-cli and run developer experience", async function() {
```
Comment (tmathern) [inline test/index.test.js:91]: Is this version unsupported now?

File: test/index.test.js [hunk 2]
```diff
@@ -116,7 +116,7 @@ describe("integration tests", function() {
         `);
     }).timeout(600000);
 
-    it("should install version 7.1.0 of aio-cli and run developer experience", async function() {
+    it.skip("should install version 7.1.0 of aio-cli and run developer experience", async function() {
```
Comment (tmathern) [inline test/index.test.js:119]: Is this version unsupported now?
If so, rather than skip we should remove it.

**Review by tmathern — CHANGES_REQUESTED** [review]

File: test/index.test.js [hunk 3]
```diff
@@ -63,13 +63,13 @@ describe("integration tests", function() {
 
     it("should install lastest version of tools and run developer experience", async function() {
         shell(`
-            npm install -g @adobe/aio-cli
+            npm install -g @adobe/aio-cli@9.4.1
```
Comment (tmathern) [inline test/index.test.js:66]: If we lock, it's not installing latest anymore...

**Comment by purplecabbage** [comment]
Please merge this and we will deal with adding tests for latest/latest in another pr.
We have daily tests that are failing until this is merged.

**Review by sudheendrakatikar — APPROVED** [review]

**Review by tmathern — APPROVED** [review]

**Comment by tmathern** [comment]
> Please merge this and we will deal with adding tests for latest/latest in another pr. We have daily tests that are failing until this is merged.

We'll remind you of that ;).
PR is approved.

**Comment by pheenomenon** [comment]
@purplecabbage Is there a followup jira you have to fix on latest aio version.

**Comment by shazron** [comment]
@pheenomenon added https://jira.corp.adobe.com/browse/ACNA-2827

@tmathern @pheenomenon Can this be squashed/merged in so I can rebase off it for PR #40 ? Thanks!

**Pull request merged**